README.md

```diff
@@ -61,7 +61,7 @@ $ git secret reveal -f
 Developing with Livepush
 ------------------------
 
-To start developing, you must first install [balenaCLI](https://github.com/balena-io/balena-cli) and set up a local-mode device.
+To start developing, you must first install [balenaCLI](https://github.com/balena-io/balena-cli) and set up a local-mode device. For development machine, you can create a virtual machine on VirtualBox following [this guide](https://www.balena.io/blog/no-hardware-use-virtualbox/), 32 GB of disk space, 4 GB RAM and 4 CPU cores are recommended. To offset workloads, you can also set up an Intel-NUC with i7 processor and 8 GB memory or more.
```

Comment:
```suggestion
To start developing, you must first install [balenaCLI](https://github.com/balena-io/balena-cli) and set up a local-mode device. This could be a virtual machine [in VirtualBox](https://www.balena.io/blog/no-hardware-use-virtualbox/) with at least 32 GB of disk space, 4 GB RAM, and 4 CPU cores. It could also be a separate device, such as an Intel NUC (i7 CPU and at least 8GB memory recommended), on your local network to move the heavy lifting off of your main machine.
```